src/main/java/org/cascadebot/cascadebot/utils/ColorUtils.java

```diff
@@ -123,6 +129,35 @@ public String getI18nMessage(Locale locale) {
 
     }
 
+    // Gets a color name based on current locale, if not found return original input
+    // This method needs a serious overhaul since I don't know how JSONConfig works...
+    private static String getColorNameFromLocale(Locale locale, String name) {
+        // Get language config
+        JSONConfig lang = Language.getLanguage(locale);
+        // Get sub config that defines the colors
+        Optional<JSONConfig> sub = lang.getSubConfig("utils.color.colors");
+        // If the sub config is not present return the name
+        if (!sub.isPresent()) {
+            return name;
+        }
+
+        // Get the sub config
+        JSONConfig colorsLangConfig = sub.get();
+        // Get all the values from the sub config
+        Map<String, Object> colors = colorsLangConfig.getValues(false);
+
+        // Loop over all values
+        for (Map.Entry<String, Object> color : colors.entrySet()) {
+            // Check if the value (color translation) is equal to the name provided
+            // Add double quotes to the name because something (valueof, jsonconfig) adds them
+            if (String.valueOf(color.getValue()).equalsIgnoreCase("\"" + name + "\"")) {
```

Comment:
Feels a bit hacky but should work reliably.

Reply:
Whattt

Reply:
I'm not proud of it but you were offline so it's actually your fault :)